test/performance/shmem_perf_suite/common.h

```diff
@@ -112,22 +112,24 @@ typedef enum {
 
 typedef enum {
     OP_FETCH,
-    OP_SET,
     OP_CSWAP,
     OP_SWAP,
     OP_FINC,
-    OP_INC,
     OP_FADD,
-    OP_ADD,
     OP_FAND,
-    OP_AND,
     OP_FOR,
-    OP_OR,
     OP_FXOR,
+    OP_SET,
+    OP_INC,
+    OP_ADD,
+    OP_AND,
+    OP_OR,
     OP_XOR,
-    SIZE_OF_OP
 } atomic_op_type;
 
+#define FIRST_FETCH_OP OP_FETCH
+#define LAST_FETCH_OP OP_FXOR
+#define LAST_OP OP_XOR
```

Comment:
What you have here is ok and doesn't need to be changed.  Just a style comment. 
 Some C coders like to include `LAST_OP` as the last item in the enum so that it is more likely to stay in sync.  Then the loop bounds is `i < LAST_OP` instead of `<=`.  You can also set `LAST_FETCH_OP` to `LAST_OP` if you made that change (but you would need to update the loop bounds check).